src/microbe_stage/Microbe.cs

```diff
@@ -1084,6 +1084,8 @@ public override void _Process(float delta)
             // Play movement sound if one isn't already playing.
             if (!movementAudio.Playing)
                 movementAudio.Play();
+
+            movementAudio.UnitDb = GD.Linear2Db(totalMovement.Length() - 15.0f) * 8.0f - 230.0f;
```

Comment:
Regarding the math here, it would be nice if you could make constants like minimum movement volume, maximum movement volume (this should be capped at like +3 Db to not allow ridiculously loud movement sound), and also then required cell speed for max volume.
Then the formula would be to calculate the percentage the cell is going from the max speed and just calculate the Db value from that.
That way the formula could be adjusted easily instead of me not having any idea about these magic numbers used here.

Reply:
When I pushed it to +80dB, it was not even so unbelievably loud.

Reply:
Still would be nice to make this "properly" with constants, and limiting the upper bound, in case godot gets some feature in the future where big Db values actually apply.